README.md

```diff
@@ -298,6 +298,16 @@ naptha personas persona_name -p "description='Persona description' module_url='i
 naptha personas -d persona_name
 ```
 
+## Toolsets
+Specially marked repos can be ingested directly to be made available to agents. These repos are loaded onto the active node and ran there.
+
+* load from hub: `naptha toolset -lh test toolset:test_toolset`
```

Comment:
The command for doing this now would be `naptha create toolset:test_toolset` (see [here](https://github.com/NapthaAI/naptha-sdk/blob/main/README.md?plain=1#L153))